AsyncDisplayKit/ASMapNode.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <MapKit/MapKit.h>
+@interface ASMapNode : ASControlNode
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
+
+@property (nonatomic, readonly) ASImageNode *mapImage;
+@property (nonatomic, readonly) ASDisplayNode *liveMap;
+/**
+ Whether the map snapshot should turn into a MKMapView when tapped on. Defaults to YES.
+ */
+@property (nonatomic, assign) BOOL hasLiveMap;
```

Comment:
@appleguy I prefer the first suggestion  "liveMap / getter = isLiveMap" this way it's super clear to the developer what is happening. Will have a look at this, this afternoon.